packages/block-editor/src/components/block-popover/index.js

```diff
@@ -69,8 +69,16 @@ function BlockPopover(
 		);
 		observer.observe( selectedElement, { attributes: true } );
 
+		const zoomedOutObserver = new window.MutationObserver(
+			forceRecomputePopoverDimensions
+		);
+		zoomedOutObserver.observe( selectedElement.closest( 'html' ), {
+			attributes: true,
+		} );
```

Comment:
```suggestion
		// The iframe of the editor canvas may be shifted by JS-based animations
		// necessitating a need to recompute the positioning of the toolbar to account
		// for the movement. The editor canvas's `html` tag is where the attributes are
		// modified during animation so by observing changes here it's possible to force
		// the toolbar to recompute as required.
		zoomedOutObserver.observe( selectedElement.closest( 'html' ), {
			attributes: true,
		} );
```

Reply:
We're really eager with these comments lately. In this case the new code is next to the old code doing the same thing. If the old code was understandable I think the new one is too. The name of the variable is also an indicator of why the observer is there 🤷🏻